package.json

```diff
@@ -23,7 +23,8 @@
     "lint": "lerna run --parallel lint",
     "bundle": "lerna run --concurrency 1 bundle",
     "test": "lerna run --concurrency 1 test",
-    "clean": "rm -frv node_modules && lerna clean --yes && lerna run --parallel clean"
+    "clean": "rm -frv node_modules && lerna clean --yes && lerna run --parallel clean",
```

Comment:
I think you should use rimraf to make it work on Windows.

Reply:
Good point, I changed it now. Thanks for pointing this out.